crates/admin/Cargo.toml

```diff
@@ -28,18 +28,18 @@ restate-wal-protocol = { workspace = true }
 anyhow = { workspace = true }
 arc-swap = { workspace = true }
 arrow-flight = { workspace = true }
-axum = { workspace = true }
+axum = { version = "0.7.5" }
 bytes = { workspace = true }
 bytestring = { workspace = true }
 codederror = { workspace = true }
 datafusion = { workspace = true }
 derive_builder = { workspace = true }
 derive_more = { workspace = true }
 futures = { workspace = true }
-http = { workspace = true }
-http-1 = { version = "1.0", package = "http" }
-hyper = { workspace = true, features = ["full"] }
-okapi-operation = { version = "0.2.2", features = ["axum-integration"] }
+http = "1.1.0"
```

Comment:
I will defer this change after the current train of open PRs gets merged. 

I avoided to commit this change up to this point, because otherwise I would needed a single PR changing all the code linking to `http` to `http-0-1`, just to change it shortly after, PR probably hard to merge because of the amount of conflicts. This intermediate state where you don't know whether `http` is `1` or `0.1` will soon end, after this and Jack PRs will be merged :)